### PR TITLE
Add some more points on heading tag and div tags.

### PR DESCRIPTION
**Pull request by ghost**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by ZedRishu** [comment]
To earn pull request I made some changes.

File: guide/english/html/index.md [hunk 1]
```diff
@@ -42,8 +42,9 @@ title: The element specifies a title for the document
 body: The element contains the visible page content
 
 h1: The element defines a large heading
-
+There are 6 Heading Tag in HTML depending on their inportance.
 p: The element defines a paragraph
+div: div tags are used to modify your html commands.
```
Comment (jonathan-grah) [inline guide/english/html/index.md]: `div` tags aren't being used in this example
```suggestion
```

File: guide/english/html/index.md [hunk 2]
```diff
@@ -42,8 +42,9 @@ title: The element specifies a title for the document
 body: The element contains the visible page content
 
 h1: The element defines a large heading
-
+There are 6 Heading Tag in HTML depending on their inportance.
```
Comment (jonathan-grah) [inline guide/english/html/index.md]: This is not the place for this description, it is better to write in a separate file in more detail about heading tags.

**Comment by jonathan-grah** [comment]
#
We are marking this PR invalid for hacktoberfest as it does not follow the recommended guidelines below:

> ### Quality Standards
> In line with Hacktoberfest value #2 (Quantity is fun, Quality is key), we have provided examples of the quality standards we encourage. This applies mainly to beginners.
> 
> - PRs that are automated e.g. scripted opening PRs to remove whitespace / optimize images.
> - PRs that are disruptive e.g. taking someone else's branch/commits and making a PR.
> - PRs that are regarded by a project maintainer as a hindrance vs. helping.
> - Something that's clearly an attempt to simply +1 your PR count for October.

Learn more at https://hacktoberfest.digitalocean.com/details

**Pull request closed without merging**